package.json

```diff
@@ -21,7 +21,7 @@
   },
   "devDependencies": {
     "chai": "*",
-    "hubot-test-helper": "akatz/hubot-test-helper",
+    "hubot-test-helper": "^1.6.0",
     "mocha": "*",
```

Comment:
Could we be good citizens and also lock these down to the specific versions that work? :+1:

Reply:
:+1: will do